packages/minuteman/extra/setup_iptables.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+# http://stackoverflow.com/questions/2870992/automatic-exit-from-bash-shell-script-on-error
+# Any subsequent(*) commands which fail will cause the shell script to exit immediately
+
+RULE="POSTROUTING -m ipvs --ipvs --vdir ORIGINAL --vmethod MASQ -m comment --comment Minuteman-IPVS-IPTables-masquerade-rule -j MASQUERADE"
+
+if ! iptables --wait -t nat -C ${RULE}; then
+  echo "Minuteman IPTables Rule not found: ${RULE}"
+  echo "Inserting Rule"
+  iptables --wait -t nat -I ${RULE} && echo "Rule successfully inserted"
+else
+  echo "Minuteman IPTables rule found"
```

Comment:
The commit now matches enough of those conditions IMHO.

- it deletes the previous minuteman rules (the older version that uses NFQUEUE <=1.8.x)
- is uses -C to check if a very specific rule exists: RULE="POSTROUTING -m ipvs --ipvs --vdir ORIGINAL --vmethod MASQ -m comment --comment Minuteman-IPVS-IPTables-masquerade-rule -j MASQUERADE". That rule even has a comment that would make it quite identifiable.
- if the specific rule is not there it adds the one it needs to ensure it keeps working.